insights/parsers/db2.py

```diff
@@ -1,54 +1,110 @@
 """
-DB2 commands
-============
+db2 - db2 database information
+==============================
+Parsers included in this module are:
 
-Shared parsers for parsing output of the commands of IBM DB2
+Db2DatabaseConfiguration - Command ``/usr/sbin/runuser -l  <user name>  -c 'db2 get database configuration for %s'``
+====================================================================================================================
 
-Db2ls - command ``db2ls -a -c``
--------------------------------
+Db2DatabaseManager - Command ``/usr/sbin/runuser -l  <user name>  -c 'db2 get dbm cfg'``
+========================================================================================
 """
+
 from insights.core import CommandParser
 from insights.core.exceptions import SkipComponent
 from insights.core.plugins import parser
 from insights.specs import Specs
 
 
-@parser(Specs.db2ls_a_c)
-class Db2ls(CommandParser, list):
```

Comment:
To remove existing parsers, please deprecate it first and state the version that it will be removed in the `deprecated` statement.

Reply:
@xiangce - my miss, we should not remove `Db2ls`, I added it back.